web/src/containers/Landing/Landing.js

```diff
@@ -58,8 +59,24 @@ class Landing extends Component{
 
     initATMData() {
         const hangSengATMs = HANG_SENG_DATA.data[0].Brand[0].ATM;
+        let hangSengLatLngLUT = new Object();
+        for (let i=0;i<HANG_SENG_LATLNG_DATA.length; i++){
+            let rec = HANG_SENG_LATLNG_DATA[i];
+            hangSengLatLngLUT[rec.address] = {'lat': rec.lat, 'lng': rec.lng};
+        }
+        for (let i=0;i<hangSengATMs.length; i++){
+            let rec = hangSengATMs[i];
```

Comment:
you are setting two new attributes on a local variable, not hangSengATMs[i]. Please double check.

Reply:
changes in rec is reflected in hangSengATMs[i]
checked with chrome.

See img:
LHS, app with distance calculation for hang seng bank working
RHS, testing the script in console

![image](https://user-images.githubusercontent.com/11174709/55273553-e9238e80-52c4-11e9-8d2b-53fab6f239f1.png)